model/src/main/scala/scalasvg/element/Circle.scala

```diff
@@ -0,0 +1,23 @@
+package scalasvg.element
+
+import scalasvg.attribute.generic.{Core, Styling, ConditionalProcessing}
+import scalasvg.element.category.Category
+
+/**
+  * The <circle> SVG element is an SVG basic shape, used to draw circles based on a center point and a radius.
+  * @param cx The x-axis coordinate of the center of the circle.
+  * @param cy The y-axis coordinate of the center of the circle.
+  * @param r The radius of the circle. A value lower or equal to zero disables rendering of the circle.
+  * @param pathLength The total length for the circle's circumference, in user units.
+  */
+final case class Circle(cx: String, cy: String, r: String, pathLength: Option[Number]=None, attributes: Seq[Circle.Attribute])
+    extends Element.Empty
+    with Category.BasicShape
```

Comment:
Example from the documentation for `Desc`:
```svg
<svg viewBox="0 0 10 10" xmlns="http://www.w3.org/2000/svg">
  <circle cx="5" cy="5" r="4">
    <desc>
      I'm a circle and that description is here to
      demonstrate how I can be described, but is it
      really necessary to describe a simple circle
      like me?
    </desc>
  </circle>
</svg>
```
It looks like `Empty` elements might not be used at all.

Reply:
Please, take a look at the permitted content of `circle` in the [documentation](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/circle#Usage_notes).

Reply:
Fixed